dags/synapse-by-the-numbers-dag.py

```diff
@@ -0,0 +1,160 @@
+"""This script is used to execute a query on Snowflake and report the results to a 
+Synapse table. This retrieves data describing the total amount of data hosted in Synapse,
+the number of active users last month, and the number of downloads last month."""
+
+from dataclasses import dataclass
+from datetime import date, datetime
+from typing import List
+
+import synapseclient
+from airflow.decorators import dag, task
+from airflow.models.param import Param
+from airflow.providers.snowflake.hooks.snowflake import SnowflakeHook
+from orca.services.synapse import SynapseHook
+
+dag_params = {
+    "snowflake_conn_id": Param("SNOWFLAKE_SYSADMIN_PORTAL_RAW_CONN", type="string"),
+    "synapse_conn_id": Param("SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN", type="string"),
+    "current_date": Param(date.today().strftime("%Y-%m-%d"), type="string"),
```

Comment:
I might make a small modification here to change this to be the month that this is being run for. I'm thinking to make it easier if we need to manually run this we don't have to think about putting the month after the one we want the data for.

So get today's date and subtract a month in lieu of it being passed in, then you won't need to use the DATEADD procedure in the sql query.

Reply:
It's a little unfortunate since I set the standard for the other queries, but thinking back I didn't really like that design so much

Reply:
Agreed. Will do!